deploy/deploy-market.ts

```diff
@@ -20,7 +20,7 @@ export const deploy: DeployFunction = async (environment) => {
     hre,
     feeWallet,
     feePercentage: 15,
-    priceMultiple: 100,
+    priceMultiple: 2000,
```

Comment:
this sets the price multiple to 20.  I don't think it hurts to set that as a default now that we're preparing to use USDC as the purchasing token.  this way tests are using a non-1 multiple value too